builtin/mainmenu/tab_credits.lua

```diff
@@ -26,38 +26,39 @@ local core_developers = {
 	"ShadowNinja <shadowninja@minetest.net>",
 	"Nathanaël Courant (Nore/Ekdohibs) <nore@mesecons.net>",
 	"Loic Blot (nerzhul/nrz) <loic.blot@unix-experience.fr>",
-	"Matt Gregory (paramat)",
-	"est31 <MTest31@outlook.com>",
-	"Craig Robbins (Zeno) <craig.d.robbins@gmail.com>",
+	"paramat",
+	"Craig Robbins (Zeno) <craig.d.robbins@gmail.com> (Dancing Queen)",
```

Comment:
Untrue statement. You might feel like a queen and dance but this doesn't make you to a queen.